instrumentation/struts-2/src/main/java/io/opentelemetry/javaagent/instrumentation/struts2/Struts2InstrumentationModule.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.struts2;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.tooling.InstrumentationModule;
+import io.opentelemetry.javaagent.tooling.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class Struts2InstrumentationModule extends InstrumentationModule {
+
+  public Struts2InstrumentationModule() {
+    super("struts-2");
```

Comment:
based on recent discussion in #1579
```suggestion
    super("struts", "struts-2.0");
```

Reply:
Renamed struts-2 to struts-2.3 everywhere.